ci/kokoro/docker/publish-refdocs.sh

```diff
@@ -65,6 +65,11 @@ echo "================================================================"
 echo "Installing docuploader package $(date)."
 python3 -m pip install --user gcp-docuploader
 
+
+echo "================================================================"
+echo "Upgrading protobuf package $(date)."
+python3 -m pip install --user protobuf --upgrade
```

Comment:
Can we do both of these in a single pip command?

Reply:
Yes, done!